### PR TITLE
Add Moto G4/G4 Plus (athene)

### PR DESCRIPTION
**Pull request by Lyudmila17**

### CODE REVIEW

**Comment by jbruechert** [comment]
Duplicate of #51

**Comment by lnjX** [comment]
But #51 didn't open an project-management issue, so this is theoretically the 'official' port.

File: manifests/motorola_athene.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+            fetch="https://github.com/Lyudmila17"
+            revision="halium-7.1" />
+
+    <project path="device/motorola/athene" name="android_device_motorola_athene" remote="lyud" />
+
+    <project path="kernel/motorola/msm8952" name="android_kernel_motorola_msm8952" remote="lyud" />
+
+    <project path="vendor/motorola/athene" name="vendor_motorola_athene" remote="lyud" />
+
+    <project path="external/bluetooth/bluedroid" name="android_external_bluetooth_bluedroid" remote="los" revision="cm-12.1" />
```
Comment (lnjX) [inline manifests/motorola_athene.xml]: Don't use cm-12.1 in halium-7.1

Comment (Lyuu17) [reply manifests/motorola_athene.xml]: That's the latest branch as far I know.

Comment (Herrie82) [reply manifests/motorola_athene.xml]: @Lyudmila17  You shouldn't need the bluedroid one for a successful build. I was able to build Athene before without bluedroid on 7.1. I don't have Athene to test myself, so I cannot comment on the state of Bluetooth without it.

Comment (lnjX) [reply manifests/motorola_athene.xml]: If the repository has no cm-14.1 branch, it's not used by any cm-14.1 device - so you don't need it then.

**Comment by Lyuu17** [comment]
libbt-vendor (bt-caf) needs some headers located on bluedroid repo. @Herrie82 how you did it compile then?

**Comment by Herrie82** [comment]
@lyudmila17 It's been a while since I touched Athene, let me try a local build again and report back. I did this for a LuneOS port for one of the users, we are still using our own manifest, so there could be some differences between what you're using and what we are using at LuneOS.

**Comment by Herrie82** [comment]
@Lyudmila17: Seems that you might want to add android_system_bt, that holds the required bits now. Like we did @ LuneOS: https://github.com/webOS-ports/android/blob/luneos-halium-7.1/default.xml#L231

**Comment by jbruechert** [comment]
Btw, you can probably reuse @Herrie82 kernel changes for athene https://github.com/herrie82/android_kernel_motorola_msm8952

**Comment by Herrie82** [comment]
@JBBgameich @Lyudmila17 These have been untested, so be aware! I just know that it builds, but no build was tested due to issues with packaging dt on LuneOS side so far. I.e. I doubt I even ran the verify-kernel script from Mer!

File: manifests/motorola_athene.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+            fetch="https://github.com/Lyudmila17"
+            revision="halium-7.1" />
+
+    <project path="device/motorola/athene" name="android_device_motorola_athene" remote="lyud" />
+
+    <project path="kernel/motorola/msm8952" name="android_kernel_motorola_msm8952" remote="lyud" />
+
+    <project path="vendor/motorola/athene" name="vendor_motorola_athene" remote="lyud" />
+
+    <project path="android_system_bt" name="android_system_bt" remote="los" revision="cm-14.1" />
```
Comment (lnjX) [inline manifests/motorola_athene.xml]: the path is wrong: should be `system/bt`

**Comment by lnjX** [comment]
@Herrie82 But even if you fix something the manifest probably won't change. Also in this repo it's about collecting all manifests, not having a list of all supported, working devices. The only condition is that it builds.

**Pull request merged**